src/client/containers/add-batch-page/add-batch-page.component.js

```diff
@@ -1,18 +1,19 @@
 import React from 'react';
 import SidebarMenu from '../../components/side-navigation/sidebar.component';
 import AddBatch from '../../components/add-batch/add-batch.component';
-import AddBatchImage from '../../components/add-batch-image/add-batch-image.component';
 import Footer from '../../components/footer/footer.component';
+import './add-batch-page.style.css';
 
 const AddBatchPage = () => {
   return (
     <>
-      <div>
+      <div className="batch-page-nav">
         <SidebarMenu isActive={true} isUser={true} />
```

Comment:
`isActive` props should be `false` for Add Batch page (it's only true for Chart Detail Page).